DRODLib/NetInterface.h

```diff
@@ -221,6 +222,8 @@ class CNetInterface {
 	// Notices
 	virtual void SetLastNotice(const UINT /*dwNoticeID*/) {}
 	virtual void QueryNotices(vector<CNetNotice> &/*notices*/, UINT /*typeMask*/, UINT /*lastId*/) const {}
+	virtual void SetDownloadHold(const int /*dwServerHoldId*/) {}
```

Comment:
Not having seen the implementation myself yet, is there a reason to use `int` with the new methods instead of `UINT`?

Reply:
Ah.  I used int because it *is* a signed int, beta holds have a negative id.  But in other places we do cast this to UINT, so I'll just update it to store it that way.